wrapper.py

```diff
@@ -310,12 +308,49 @@ def _load_custom_shape(f):
     """
     data_reshaped = []
     counter = 0
-    for l in f.attrs['oldshape']:
-        data_reshaped.append(np.array(f.value[counter:counter + l]))
+    for i in xrange(len(f.attrs['oldshape'])):
```

Comment:
I merged the master into this branch. We here need a `zip()` now because the value_types are stored in addition and have to be retrieved.
I also renamed the variables `counter,l` because I didn't see a reason for the previous names since these are simply indices.